src/components/PeopleFilters.tsx

```diff
@@ -1,21 +1,68 @@
-export const PeopleFilters = () => {
+import classNames from 'classnames';
+import { SearchLink } from './SearchLink';
+import { CenturiesOptions, SexOptions } from '../utils/data';
+
+type Props = {
+  sex: SexOptions;
+  centuries: CenturiesOptions[];
+  query: string;
+  handleQueryChange: (event: React.ChangeEvent<HTMLInputElement>) => void,
+};
+
+export const PeopleFilters: React.FC<Props> = ({
+  sex,
+  centuries,
+  query,
+  handleQueryChange,
+}) => {
+  const centuryParams = (century: CenturiesOptions) => {
+    return centuries.includes(century)
+      ? centuries.filter(num => century !== num)
+      : [...centuries, century];
+  };
+
+  const resetParams = {
+    centuries: null,
+    sex: null,
+    query: null,
+  };
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">All</a>
-        <a className="" href="#/people?sex=m">Male</a>
-        <a className="" href="#/people?sex=f">Female</a>
+        <SearchLink
+          className={classNames({ 'is-active': !sex })}
+          params={{ sex: null }}
+        >
+          All
+        </SearchLink>
+
+        <SearchLink
+          className={classNames({ 'is-active': sex === SexOptions.MALE })}
+          params={{ sex: SexOptions.MALE }}
+        >
+          Male
+        </SearchLink>
+
+        <SearchLink
+          className={classNames({ 'is-active': sex === SexOptions.FEMALE })}
+          params={{ sex: SexOptions.FEMALE }}
+        >
+          Female
+        </SearchLink>
       </p>
 
       <div className="panel-block">
         <p className="control has-icons-left">
           <input
             data-cy="NameFilter"
-            type="search"
+            type="text"
```

Comment:
The input type should be 'search', not 'text'. The 'search' type is specially designed to make search boxes more user-friendly.